src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1227,6 +1227,200 @@ public void clickRemoveMsqOptionLinkForNewQuestion(int optionIndex) {
+        assertTrue(isMsqMinSelectableChoicesEnabled(qnNumber));
+
+        WebElement inputBox = getMsqMinSelectableChoicesBox(qnNumber);
+        String id = inputBox.getAttribute("id");
+
+        executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
+    }
+
+    public void setMsqMaxSelectableChoices(int qnNumber, int value) {
+        assertTrue(isMsqMaxSelectableChoicesEnabled(qnNumber));
+
+        WebElement inputBox = getMsqMaxSelectableChoicesBox(qnNumber);
+        String id = inputBox.getAttribute("id");
+
+        executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
+    }
+
+    public void fillMsqOption(int qnNumber, int optionIndex, String optionText) {
+        WebElement optionBox = browser.driver.findElement(By.id("msqOption-" + optionIndex + "-" + qnNumber));
+        fillTextBox(optionBox, optionText);
+    }
+
+    public int getNumOfStudentsForFs() {
+        return Integer.parseInt(browser.driver.findElement(By.id("num-students")).getAttribute("value"));
+    }
+
+    public int getNumOfTeamsForFs() {
+        return Integer.parseInt(browser.driver.findElement(By.id("num-teams")).getAttribute("value"));
+    }
+
+    public int getNumOfInstructorsForFs() {
+        return Integer.parseInt(browser.driver.findElement(By.id("num-instructors")).getAttribute("value"));
+    }
+
+    private String getValue(WebElement elem) {
+        return (String) executeScript(String.format("return $('#%s').val();", elem.getAttribute("id")));
+    }
+
+    public int getMsqMinSelectableChoices(int qnNumber) {
+        assertTrue(isMsqMinSelectableChoicesEnabled(qnNumber));
+
+        return Integer.parseInt(getValue(getMsqMinSelectableChoicesBox(qnNumber)));
+    }
+
+    public int getMsqMaxSelectableChoices(int qnNumber) {
+        assertTrue(isMsqMaxSelectableChoicesEnabled(qnNumber));
+
+        return Integer.parseInt(getValue(getMsqMaxSelectableChoicesBox(qnNumber)));
+    }
+
+    private boolean isGenerateOptionsForMsqChecked(int qnNumber) {
+        return browser.driver.findElement(By.id("generateMsqOptionsCheckbox-" + qnNumber)).isSelected();
+    }
+
+    private String getGenerateOptionsForMsqValue(int qnNumber) {
+        Select options = new Select(browser.driver.findElement(By.id("msqGenerateForSelect-" + qnNumber)));
+
+        return options.getFirstSelectedOption().getText();
+    }
+
+    public int getNumOfMsqOptions(int qnNumber) {
+        if (isGenerateOptionsForMsqChecked(qnNumber)) {
+            String selectedOption = getGenerateOptionsForMsqValue(qnNumber);
+
+            switch (selectedOption) {
+            case "students":
+                return getNumOfStudentsForFs();
+            case "teams":
+                return getNumOfTeamsForFs();
+            case "instructors":
+                return getNumOfInstructorsForFs();
+            default:
+                return 0;
+            }
+        } else {
```

Comment:
This `else` block is not needed. The subsequent `return` can simply be outside the `if` block.